installer/templates/new/priv/gettext/errors.pot

```diff
@@ -0,0 +1,100 @@
+#################################################################
+#
+# # Gettext Usage
+#
+# ## .pot files
+#
+#   .pot files hold your translation msgid's for Gettext
+#
+#   .pot entries are *extracted from source automatically*.
+#   You should only add msgid entries to .pot files for dynamic
+#   keys that are not compile-time literals, such as Ecto msgid's.
+#   The msgstr's for .pot entries will always be blank strings.
+#
+# ## .po files
+#
+#   Translations for the msgid's defined in .pot files go in the
+#   corresponding .po file, located in:
+#
+#       /priv/gettext/[locale]/LC_MESSAGES/[domain].po
+#
+#   where `locale` could be "en" and `domain` "default", "errors"
+#
+#   For .po files, you only enter the msgstr translations.
+#   The msgid's are extracted and merged into the po files for you.
+#
+# See http://hexdocs.pm/gettext for detailed docs
+#################################################################
+
+# Ecto validations
```

Comment:
Let's add the name of the validation function on top of each of them? Like "From Ecto.Changeset.cast/4", "From Ecto.Changeset.validate_length/3" and so on?